client/src/components/reader/SectionPagination.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component, PropTypes } from 'react';
+
+export default class SectionPagination extends Component {
+
+  static propTypes = {
+    text: PropTypes.object,
+    textId: PropTypes.string,
+    sectionId: PropTypes.string
+  };
+
+  getSiblingSection(id, shift) {
+    let siblingSection = false;
+    this.props.text.attributes.toc.forEach((section, i) => {
+      if (section.id === id && this.props.text.attributes.toc[i + shift]) {
+        siblingSection = this.props.text.attributes.toc[i + shift];
+      }
+    });
+
+    return siblingSection;
+  }
+
+  getSectionPath(section) {
+    let path = '';
+    let anchor = '';
+    if (section.anchor) anchor = `#${section.anchor}`;
+    path = `/read/${this.props.text.id}/section/${section.id}${anchor}`;
+    return path;
+  }
+
+  getPreviousLink() {
+    let previousLink = '';
+    const previousNode = this.getSiblingSection(Number(this.props.sectionId), - 1);
+    if (previousNode) {
+      const previousPath = this.getSectionPath(previousNode);
+      previousLink = (
+        <a href={previousPath} className="pagination-previous">
```

Comment:
I merged this without catching this—what you're doing here will cause a full page reload. For links, we need to use the <Link> component provided by React Router, as we do in the reader TOC.

I'll fix in a subsequent commit.